cg/constants/constants.py

```diff
@@ -97,15 +102,23 @@ class PrepCategory(StrEnum):
     WHOLE_TRANSCRIPTOME_SEQUENCING: str = "wts"
 
 
-PREP_CATEGORIES = ("cov", "mic", "rml", "tgs", "wes", "wgs", "wts")
+class SexOptions(StrEnum):
```

Comment:
Yeah, I saw that.. However the PhenotypeStatus has an extra value of an empty string. It would be a change in logic to allow it in the database as well. Will leave that for a future PR